src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -3,6 +3,8 @@ import environment from '../../../../platform/utilities/environment';
 import FormFooter from '../../../../platform/forms/components/FormFooter';
 import preSubmitInfo from '../../../../platform/forms/preSubmitInfo';
 
+import services from 'platform/monitoring/DowntimeNotification/config/externalServices';
```

Comment:
Looks like you can just import `externalServices` from `platform/monitoring/DowntimeNotification`